log/handler.go

```diff
@@ -277,13 +277,13 @@ func builtinReplace(_ []string, attr slog.Attr) slog.Attr {
 		if v == nil {
 			attr.Value = slog.AnyValue("<nil>")
 		} else {
-			attr.Value = slog.AnyValue(formatLogfmtBigInt(v))
+			attr.Value = slog.AnyValue(v.String())
 		}
 	case *uint256.Int:
 		if v == nil {
 			attr.Value = slog.AnyValue("<nil>")
 		} else {
-			attr.Value = slog.AnyValue(formatLogfmtUint256(v))
+			attr.Value = slog.AnyValue(v.ToBig().String())
```

Comment:
`AnyValue` -> `StringValue`. And use `v.Dec`, so `attr.Value = slog.StringValue(v.Dec())` instead

Reply:
if we use `StringValue`, the value will be quoted in the output.

Reply:
RE relevant comment below:  I was mistaken.  To avoid values in the output getting quoted we must set the value to `slog.AnyValue(fmt.Stringer)`.  so we would need to wrap `uint256.Int` in a custom type to make the default string format decimal and not hex.

Reply:
Hm. This is what I have
```
	switch v := attr.Value.Any().(type) {
	case time.Time:
		attr = slog.Any(attr.Key, v.Format(timeFormat))
	case *uint256.Int:
		if v == nil {
			attr.Value = slog.StringValue("<nil>")
		} else {
			attr.Value = slog.StringValue(v.Dec())
		}
	}
```
And it spits out this: 
```
$ go run -tags=integrationtests ./cmd/geth --log.format=logfmt logtest 
t=2023-11-08T12:41:23+0100 lvl=info msg=big.Int 111,222,333,444,555,678,999=111222333444555678999
t=2023-11-08T12:41:23+0100 lvl=info msg=-big.Int -111,222,333,444,555,678,999=-111222333444555678999
t=2023-11-08T12:41:23+0100 lvl=info msg=big.Int 11,122,233,344,455,567,899,900=11122233344455567899900
t=2023-11-08T12:41:23+0100 lvl=info msg=-big.Int -11,122,233,344,455,567,899,900=-11122233344455567899900
t=2023-11-08T12:41:23+0100 lvl=info msg=uint256 111,222,333,444,555,678,999=111222333444555678999
t=2023-11-08T12:41:23+0100 lvl=info msg=uint256 11,122,233,344,455,567,899,900=11122233344455567899900

```

No problematic quoting.. ?

Reply:
no.  I'm completely wrong.  it appears only byte-slices get quoted. https://cs.opensource.google/go/x/exp/+/7918f672:slog/text_handler.go;l=114